internal/repotesting/repotesting.go

```diff
@@ -48,7 +48,7 @@ func (e *Environment) Setup(t *testing.T, opts ...func(*repo.NewRepositoryOption
 		BlockFormat: content.FormattingOptions{
 			HMACSecret: []byte{},
 			Hash:       "HMAC-SHA256",
-			Encryption: encryption.NoneAlgorithm,
```

Comment:
Not sure how large of an impact this will have in tests running time.
The initial thinking was to allow this for testing if needed.

Reply:
if this is too slow we can always go back to unencrypted. I don't see any measurable difference.